### PR TITLE
DelayPacket and DelayMessage don't use RequestCache anymore, initial

### PR DESCRIPTION
**Pull request by NielsZeilemaker**
commit

### CODE REVIEW

**Comment by NielsZeilemaker** [comment]
@whirm ready to be merged

**Comment by NielsZeilemaker** [comment]
@whirm ready to be merged

**Comment by whirm** [comment]
@NielsZeilemaker Could you rebase it?

**Comment by NielsZeilemaker** [comment]
@whirm done

File: community.py [hunk 1]
```diff
@@ -1815,6 +1796,95 @@ def _generic_timeline_check(self, messages):
...
+            now = time()
+            for delayed in self._delayed_value.keys():
+                if now > delayed.timestamp + 10:
+                    self._remove_delayed(delayed)
+                    delayed.on_timeout()
+
+                    self.dispersy.statistics.delay_timeout += 1
+
+                    self._dispersy._statistics.drop_count += 1
+                    self._dispersy._statistics.dict_inc(self._dispersy._statistics.drop, "delay_timeout:%s" % delayed)
```
Comment (whirm) [inline community.py]: 120?

File: community.py [hunk 2]
```diff
@@ -1815,6 +1796,95 @@ def _generic_timeline_check(self, messages):
...
+            self._dispersy._statistics.dict_inc(self._dispersy._statistics.delay, "delay_packet:%s" % delay)
+            delay.delayed = packet
+            delay.candidate = candidate
+
+    def _resume_delayed(self, meta, messages):
+        has_mid = isinstance(meta.authentication, (MemberAuthentication, DoubleMemberAuthentication))
+        has_seq = isinstance(meta.distribution, FullSyncDistribution) and meta.distribution.enable_sequence_number
+
+        received_keys = [(meta.name, message.authentication.member.mid if has_mid else None,
+                       message.distribution.global_time, message.distribution.sequence_number if has_seq else None) for message in messages]
```
Comment (whirm) [inline community.py]: 120?

**Comment by whirm** [comment]
Looks OK to me, could you trim the long lines a bit? (the ones you modified)

**Comment by NielsZeilemaker** [comment]
@whirm done

**Pull request merged**